src/5.0_to_5.1/migrate_5.0_to_5.1.ts

```diff
@@ -1,12 +1,38 @@
 import { MigrationFunction } from "../migration.types";
 import { migrateDashboard } from "./migrateDashboard";
 import { migrateWidget } from "./migrateWidget";
+import { getNamesOfCalculatedMeasuresToMigrate } from "./calculated-measures/getNamesOfCalculatedMeasuresToMigrate";
+import { migrateSavedCalculatedMeasures } from "./calculated-measures/migrateSavedCalculatedMeasures";
 
 export const migrate_50_to_51: MigrationFunction = (
   contentServer,
-  { migrateDashboards, migrateSavedWidgets },
+  {
+    migrateDashboards,
+    migrateSavedWidgets,
+    errorReport,
+    counters,
+    doesReportIncludeStacks,
+  },
 ) => {
-  // TODO migrate calculated measures
+  const namesOfCalculatedMeasuresToMigrate =
+    getNamesOfCalculatedMeasuresToMigrate(contentServer);
+  // Accumulate the cubes to which the saved calculated measures belong.
+  const measureToCubeMapping: { [measureName: string]: string[] } = {};
+
   migrateDashboards(migrateDashboard);
```

Comment:
Ultimately we'll need to wire the variables related to calculated measures here as well.
But I prefer to do it in the separate PR where we make the whole thing work (and factorizing the mutation logic between `migrateWidget` and `migrateDashboard`).